src/ontology/mondo-edit.obo

```diff
@@ -418920,6 +418920,7 @@ xref: UMLS:C4539903 {source="OMIM:617610", source="MONDO:notFoundInDiseaseSubset
 is_a: MONDO:0020642 {source="OMIM:617610"} ! polycystic kidney disease
 intersection_of: MONDO:0020642 ! polycystic kidney disease
 intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/26551 ! DZIP1L
+intersection_of: has_modifier HP:0000007 ! Autosomal recessive inheritance
```

Comment:
which DP does this conform to?

We typically do as hidden GCIs

Reply:
note to self - we created a new pattern for this